Xamarin.Forms.Platform.UAP/LabelRenderer.cs

```diff
@@ -312,5 +315,17 @@ void UpdateLineHeight(TextBlock textBlock)
 				textBlock.LineHeight = Element.LineHeight * textBlock.FontSize;
 			}
 		}
+
+		void UpdateMaxLines(TextBlock textBlock)
+		{
+			if (Element.MaxLines >= 0)
+			{
+				textBlock.MaxLines = Element.MaxLines;
+			}
+			else
+			{
+				textBlock.MaxLines = 0;
```

Comment:
0 means 'no maximum', right ?

Reply:
0 == auto and is the default value: https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.textblock.maxlines#Windows_UI_Xaml_Controls_TextBlock_MaxLines